docs/form-elements/datetime.md

```diff
@@ -6,18 +6,21 @@
+| Property                 | Required | Type      | Default      | Description                                                                                                                                                                                                                                                                                            |
+| ------------------------ | -------- | --------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `type`                   | Yes      | `string`  | `'datetime'` | The type of Form Element.                                                                                                                                                                                                                                                                              |
+| `name`                   | Yes      | `string`  |              | The key that will be assigned a value in the submission data when the form is submitted.                                                                                                                                                                                                               |
+| `label`                  | Yes      | `string`  |              | Display text presented to the user above the input by default.                                                                                                                                                                                                                                         |
+| `hint`                   | No       | `string`  |              | A hint triggered by an icon tooltip to be displayed when hovering beside the element label.                                                                                                                                                                                                            |
+| `defaultValue`           | No       | `string`  |              | A default value when the form is opened. Must be in [ISO_8601 Timestamp](https://en.wikipedia.org/wiki/ISO_8601) format, or the string 'NOW' to denote that `defaultValueDaysOffset` will also be used to dynamically offset the `defaultValue` from the time the form is loaded.                      |
+| `defaultValueDaysOffset` | No       | `number`  |              | This value must be set if `defaultValue` is set to `NOW`. It can be a negative or positive number.                                                                                                                                                                                                     |
```

Comment:
```suggestion
| `defaultValueDaysOffset` | No       | `number`  |              | This value can only be set if `defaultValue` is `NOW`. It can be a negative or positive number.                                                                                                                                                                                                     |
```